users/userRouter.js

```diff
@@ -1,47 +1,106 @@
-const express = require('express');
+const express = require("express");
+const Users = require("./userDb");
+const Posts = require("../posts/postDb");
 
 const router = express.Router();
 
-router.post('/', (req, res) => {
-  // do your magic!
+router.post("/", validateUser, validatePost, (req, res, next) => {
+  Users.insert({ name: req.body.name })
+    .then(user => {
+      res.status(200).json(user);
+    })
+    .catch(next);
 });
 
-router.post('/:id/posts', (req, res) => {
-  // do your magic!
+router.post("/:id/posts", validateUserId, (req, res, next) => {
```

Comment:
This is the Users Router but you have a Posts operation. This looks like it belongs in the postRouter.js